### ISSUE TITLE
LLVM 16.0.5 fails to install on GitHub Actions `ubuntu-latest` runners

### ISSUE THREAD

**Issue by brandtbucher** [KyleMayes/install-llvm-action#54]
It appears that the expected download URL (https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/clang+llvm-16.0.5-x86_64-linux-gnu-ubuntu-22.04.tar.xz) doesn't exist.

```
2023-07-03T20:59:01.3253451Z ##[group]Run KyleMayes/install-llvm-action@v1
2023-07-03T20:59:01.3253793Z with:
2023-07-03T20:59:01.3254013Z   version: 16
2023-07-03T20:59:01.3254288Z   directory: /home/runner/work/_temp/llvm
2023-07-03T20:59:01.3254555Z env:
2023-07-03T20:59:01.3254856Z   pythonLocation: /opt/hostedtoolcache/Python/3.10.12/x64
2023-07-03T20:59:01.3255258Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.10.12/x64/lib/pkgconfig
2023-07-03T20:59:01.3255639Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.12/x64
2023-07-03T20:59:01.3256011Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.12/x64
2023-07-03T20:59:01.3256541Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.12/x64
2023-07-03T20:59:01.3256903Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.10.12/x64/lib
2023-07-03T20:59:01.3257178Z ##[endgroup]
2023-07-03T20:59:01.4219161Z Installing LLVM and Clang 16 (16.0.5)...
2023-07-03T20:59:01.4225610Z Downloading and extracting 'https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/clang+llvm-16.0.5-x86_64-linux-gnu-ubuntu-22.04.tar.xz'...
2023-07-03T20:59:01.6716004Z Error: Unexpected HTTP response: 404
2023-07-03T20:59:01.6716684Z     at /home/runner/work/_actions/KyleMayes/install-llvm-action/v1/dist/index.js:4214:25
2023-07-03T20:59:01.6717084Z     at Generator.next (<anonymous>)
2023-07-03T20:59:01.6717615Z     at fulfilled (/home/runner/work/_actions/KyleMayes/install-llvm-action/v1/dist/index.js:4104:32)
2023-07-03T20:59:01.6718124Z     at processTicksAndRejections (node:internal/process/task_queues:96:5)
2023-07-03T20:59:01.6754791Z ##[error]Unexpected HTTP response: 404
```

**Comment by Chamber6821** [comment]
I think temporary solution is to remove 16.0.5 from supported versions.

**Comment by KyleMayes** [comment]
I've released a change that fixes this, 16.0.5 and 16.0.6 weren't released for (non-aarch64) Linux and macOS so they are now indicated as only supported on Windows.

**Comment by brandtbucher** [comment]
Awesome, thanks for the quick response (and the great GitHub Action)! :)
